test/linalg/cholesky.jl

```diff
@@ -94,18 +91,12 @@ using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted, PosDefException
         U = Bidiagonal([2,sqrt(eltya(3))],[-1],true) / sqrt(eltya(2))
         @test full(chol(S)) ≈ full(U)
 
-        #lower Cholesky factor
```

Comment:
If you mean that the comment added on line 61 should also apply to this block, perhaps move this block adjacent to the block to which that comment is attached (rather than leave those blocks separated by the block dedicated to Strang matrices)? Forty lines, paragraph breaks, and other comments interceding obscures the relationship :).